pandas_gbq/gbq.py

```diff
@@ -762,6 +824,26 @@ def create(self, table_id, schema):
         table = Table(table_ref)
         table.schema = pandas_gbq.schema.to_google_cloud_bigquery(schema)
 
+        if clustering_columns:
+            table.clustering_fields = list(clustering_columns)
+
+        if time_partitioning_column:
```

Comment:
Technically someone could enable time partitioning without a column to do [ingestion time partitioning](https://docs.cloud.google.com/bigquery/docs/partitioned-tables#ingestion_time). This is less flexible than using a column, though, so I'm not sure we need to worry about it.